src/libmongoc/src/mongoc/mongoc-shared-private.h

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright 2021 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
```suggestion
#include "mongoc-prelude.h"
```

This should fix the [check-headers](https://spruce.mongodb.com/task/mongo_c_driver_releng_check_headers_patch_b80d00f0993501b21193b1869f43949d3e80a044_613910382a60ed70d40d44d6_21_09_08_19_34_18/logs?execution=0) task on Evergreen.